pkg/kindsys/report.go

```diff
@@ -71,7 +71,7 @@ var plannedCoreKinds = []string{
 	"Folder",
 	"DataSource",
 	"APIKey",
-	"ServiceAccount",
+	"Serviceaccount",
```

Comment:
@sdboyer the names need to be "Captilized" only. Not have multiple case letters, it seems like.

This could maybe be changed? 🤔 

WDYT?